install_radar_deps.py

```diff
@@ -303,102 +313,113 @@ def fix_boost_links():
-        "update-alternatives --remove-all gcc;" \
-        "update-alternatives --remove-all g++;" \
-        "update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 50;" \
-        "update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-7 50;" \
-        "update-alternatives --config gcc;" \
-        "update-alternatives --config g++;"
+                             "update-alternatives --remove-all gcc;" \
+                             "update-alternatives --remove-all g++;" \
+                             "update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 50;" \
+                             "update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-7 50;" \
+                             "update-alternatives --config gcc;" \
+                             "update-alternatives --config g++;"
         execute_cmd(pre_cuda_setup_cmd)
-        cuda_file = 'cuda_10.2.89_440.33.01_linux.run'
+        cuda_file = 'cuda_11.4.3_470.82.01_linux.run'
 
     cuda_cmd = "cd ${{IDIR}};" \
-    "wget -N http://developer.download.nvidia.com/compute/cuda/10.2/Prod/local_installers/{cuda_file};" \
-    "sh {cuda_file} --silent --toolkit --samples;".format(cuda_file=cuda_file)
+               "wget -N http://developer.download.nvidia.com/compute/cuda/10.2/Prod/local_installers/{cuda_file};" \
+               "sh {cuda_file} --silent --toolkit --samples;".format(cuda_file=cuda_file)
 
     execute_cmd(cuda_cmd)
 
+
 def install_realtime():
     """
     Create virtual environment and install utilities needed for RT capabilities.
     """
 
     rt_cmd = "bash -c \"cd /usr/local;" \
-    "git clone https://github.com/vtsuperdarn/hdw.dat.git;" \
-    "mkdir -p $BOREALISPATH/borealisrt_env;" \
-    "virtualenv -p python3.9 $BOREALISPATH/borealisrt_env;" \
-    "source $BOREALISPATH/borealisrt_env/bin/activate;" \
-    "pip install zmq;" \
-    "pip install git+git://github.com/SuperDARNCanada/backscatter.git#egg=backscatter;" \
-    "pip install pydarn;" \
-    "deactivate;\""
+             "git clone https://github.com/SuperDARN/hdw.git;" \
+             "mkdir -p $BOREALISPATH/borealisrt_env;" \
+             "virtualenv -p python3.9 $BOREALISPATH/borealisrt_env;" \
+             "source $BOREALISPATH/borealisrt_env/bin/activate;" \
+             "pip install zmq;" \
+             "pip install git+https://github.com/SuperDARNCanada/backscatter.git#egg=backscatter;" \
+             "pip install pydarn;" \
+             "deactivate;\""
 
     execute_cmd(rt_cmd)
 
+
 def install_dspenv():
     """
     Create virtual environment and install utilities needed for python DSP.
     """
 
     rt_cmd = "bash -c \"mkdir -p $BOREALISPATH/dspenv;" \
-    "virtualenv -p python3.9 $BOREALISPATH/dspenv;" \
-    "source $BOREALISPATH/dspenv/bin/activate;" \
-    "pip install zmq numpy scipy matplotlib cupy protobuf posix_ipc;" \
-    "deactivate;\""
+             "virtualenv -p python3.9 $BOREALISPATH/dspenv;" \
```

Comment:
The code should be robust enough to select or catch the python version we want to use for install. Like python3.6, 3.9, or soon 3.11.

Reply:
I hard-coded 3.9 in here as it is the minimum version sufficient for shared memory operations that we now do in the develop branch. I agree though, this would be good to have in the future.

Reply:
Did some reworking, now it is an option that you can specify when running the script. It writes this version to the .bashrc file to be read by steamed_hams.py (will be supported once PR #343 is merged) for proper operation.